src/src/utils/Constants.js

```diff
@@ -0,0 +1,5 @@
+// Auth
+export const LOG_IN = "LOG_IN";
```

Comment:
suggestion: you can create an auth object and add these constants in that object

Reply:
Ok will try that